lib/output.js

```diff
@@ -146,11 +150,10 @@ function OutputStream(options) {
               case "\b": return "\\b";
               case "\f": return "\\f";
               case "\x0B": return options.screw_ie8 ? "\\v" : "\\x0B";
-              case "\u2028": return "\\u2028";
-              case "\u2029": return "\\u2029";
-              case "\ufeff": return "\\ufeff";
               case "\0":
                   return /[0-7]/.test(str.charAt(i+1)) ? "\\x00" : "\\0";
+              default:
+                  return to_ascii(s);
```

Comment:
Sorry about that, that's a really stupid oversight on my part. I'll push a fix as soon as I can figure out how to test \u2028 properly, which now is correctly converted to a literal in ascii.js::ascii_only_false. Setting expect_exact to "...\u2028..." results in a JS_Parse_Error.